src/js/components/PackagesTable.js

```diff
@@ -46,7 +46,7 @@ class PackagesTable extends React.Component {
 
   getClassName(prop, sortBy, row) {
     return classNames({
-      'highlight': prop === sortBy.prop,
+      'table-cell-sorted': prop === sortBy.prop,
```

Comment:
The `table-cell-sorted` should probably just be replaced with `active`.  See http://localhost:4200/#/styles/content/tables/ for reference.